src/TLabel.h

```diff
@@ -39,10 +41,40 @@ Q_OBJECT
 
 public:
 
-                  TLabel( QWidget * pW=0 );
-void              setScript( Host * pHost, QString & func, TEvent * args ){ mpHost = pHost; mScript = func; mpParameters = args; }
-void              setEnter( Host * pHost, QString & func, TEvent * args ){ mpHost = pHost; mEnter = func; mEnterParams = args; }
-void              setLeave( Host * pHost, QString & func, TEvent * args ){ mpHost = pHost; mLeave = func; mLeaveParams = args; }
```

Comment:
Rather than doing the complex stuff below, just have it be `TEvent args` and leave everything else alone.